bazel/external/googleurl.patch

```diff
@@ -56,10 +56,10 @@ index 0cd36dc..8c4cbd4 100644
 # https://github.com/envoyproxy/envoy/issues/14743.
 
 diff --git a/url/BUILD b/url/BUILD
-index f2ec8da..4e2d55b 100644
+index 6ed3fc5..64b7a2b 100644
 --- a/url/BUILD
 +++ b/url/BUILD
-@@ -52,3 +52,27 @@ cc_library(
+@@ -54,3 +54,29 @@ cc_library(
          "//polyfills",
      ] + build_config.icuuc_deps,
  )
```

Comment:
Unclear what causes the compilation failure here on windows, I see nothing specific, could you merge main

Reply:
I think this PR can merge as is, and I will let @dio decide on the repo migration.
Currently the updated googleurl library has `-Wno-c++11-narrowing`.
Windows CI is throwing "cl : Command line error D8021 : invalid numeric argument '/Wno-c++11-narrowing'"

Reply:
/wait

This cannot be merged as is, see the bazel/*.bzl logic on how we toggle flags by compiler.